internal/ramses/ramses.go

```diff
@@ -0,0 +1,39 @@
+package ramses
+
+import (
+	"encoding/binary"
+	"io"
+	"time"
+)
+
+// Ramses data header
+type Ramses struct {
+	Synch  uint16 // fixed to 0xEB90
+	Length uint16 // payload length
+	Port   uint16
+	Type   uint8
+	Secure uint8
+	Time   uint32 // milliseconds since midnight
+	Date   int32  // days since 2000-01-01 00:00:00.00
+}
+
+// Read Ramses
+func (ramses *Ramses) Read(buf io.Reader) error {
+	return binary.Read(buf, binary.LittleEndian, ramses)
```

Comment:
how about a test of a real Ramses buffer from a `.rac` file resulting in the expected struct?